### DISCUSSION TITLE
PO-TABLE - Footer/Header fixos

### DISCUSSION THREAD

**Discussion by gustavo4ndrade** [po-ui/po-angular#899] [Tire suas dúvidas]
Boa noite, pessoal!
Existe alguma forma de ter uma coluna ou uma linha fixa  da tabela usando o PO-TABLE? Se não, vocês conseguem indicar um caminho?

exemplos: 
https://stackblitz.com/angular/mnmgmdeglal?file=src%2Fapp%2Ftable-sticky-complex-flex-example.ts
https://stackblitz.com/angular/egpnrdlylxo?file=src%2Fapp%2Ftable-sticky-footer-example.ts

Queria fazer uma somatória dos valores de cada coluna e deixar as 2 ultimas linhas da tabela fixadas enquanto o restante "scrolla" normalmente.

**Comment by jhosefmarks** [comment]
Atualmente não é possível!

**Comment by Gregwkk** [comment]
Nativamente não tem jeito mesmo. Mas na página da Table do Po-Ui tem uma dica que pode te ajudar. Tem um exemplo do Air Fare onde há uma linha adicional abaixo da tabela. Ela na verdade não faz parte do po-table. É um po-info que soma os valores de acordo com a seleção das linhas.  Você teria que fazer o mesmo para cada uma das situações, adicionando uma Po-info para soma de valores e outra para 2 ultimas linhas da tabela. Ou então gerar mais uma tabela nova com essas 2 linhas e manter separada da original.

![image](https://user-images.githubusercontent.com/25347640/122945064-aee3b300-d34e-11eb-92b2-702c2a862199.png)
